test/dotnet-new3.UnitTests/Helpers.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace dotnet_new3.UnitTests
+{
+    static class Helpers
+    {
+        public static string CreateTemporaryFolder([CallerMemberName] string name = "")
+        {
+            string workingDir = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString(), name);
```

Comment:
I prefer when temp folder is somehow readable. Wither file prefix $"intent-{Guid.NewGuid()}" or better "intent\\{Guid.NewGuid()}"

Reply:
Done